OpenDreamShared/Dream/Procs/ProcAttributes.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.ComponentModel;
+
+namespace OpenDreamShared.Dream.Procs;
+
+[Flags]
+[DefaultValue(None)]
+public enum ProcAttributes
+{
+    None = 1 << 0, // Internal
+    IsOverride = 1 << 1, // Internal
+    Unimplemented = 1 << 2,
```

Comment:
Is this not internal too?

Reply:
Not really? It's set in DM code by a coder. It just happens to be unique to OD.